pkg/witness/publish_checkpoint.go

```diff
@@ -151,7 +151,9 @@ func (c *CheckpointPublisher) publish(tc *trillianclient.TrillianClient, sTreeID
 
 	// return value ignored, which is whether or not the entry was set
 	// no error is thrown if the key already exists
-	successNX, err := c.redisClient.SetNX(ctx, key, hexCP, 0).Result()
+	// use smallest value as it's unused
+	value := true
+	successNX, err := c.redisClient.SetNX(ctx, key, value, 0).Result()
```

Comment:
`/latest` stores the most recent stable checkpoint and is read when requesting a stable checkpoint through `api/v1/log?stable=true` - https://github.com/sigstore/rekor/blob/ca115c94322d9b7759138ef12438118465408896/pkg/api/tlog.go#L57-L59

This key is only used to prevent multiple instances from writing different checkpoints, in the case of clock skew. If one instance were behind, this prevents it from overwriting the checkpoint. Its value is never used.